yellowbrick/classifier/confusion_matrix.py

```diff
@@ -80,6 +84,7 @@ def __init__(self, model, ax=None, classes=None, **kwargs):
         self.cmap.set_under(color='w')
         self.cmap.set_over(color='#2a7d4f')
         self.edgecolors=[] #used to draw diagonal line for predicted class = true class
+        self.label_mapping= label_mapping
```

Comment:
@bbengfort I suggested the dictionary in #244. But, I agree that it should be user choice - either the `LabelEncoder` itself OR a dictionary mapping. It could use duck typing with the same variable (label_mapping) - use a `try:` block with inverse_transform, and if `self.label_mapping` doesn't have that method (because it's not a `LabelEncoder`, treat it as a dictionary.